tornjak-frontend/src/components/navbar.tsx

```diff
@@ -39,21 +85,27 @@ class NavigationBar extends Component<NavigationBarProp, NavigationBarState> {
               <a href="/clusters" className="dropbtn">Clusters </a>
               <div className="dropdown-content">
                 <a href="/clusters" className="nav-link">Clusters List</a>
-                <a href="/cluster/clustermanagement" className="nav-link">Cluster Management</a>
+                {this.TornjakHelper.checkRolesAdminUser(this.props.globalUserRoles) &&
```

Comment:
can we make this cleaner somehow by using an initialized variable?

Reply:
done!